src/main/java/com/google/cloud/teleport/splunk/SplunkEventWriter.java

```diff
@@ -328,6 +332,7 @@ public Builder withUrl(ValueProvider<String> url) {
      */
     public Builder withUrl(String url) {
       checkArgument(url != null, "withURL(url) called with null input.");
+      checkArgument(url.startsWith("http"), "input url has no protocol.");
```

Comment:
Same as earlier comment.  Instead of doing twice move implementation to method say isValidUrlFormat(String url) returning boolean value.  
```
checkArgument(isValidUrlFormat(url),"Invalid url format. Url format should match either http://domain or https://domain");
```

Reply:
Added method `isValidUrlFormat(String url)` to validate URLs